src/components/Input.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from 'react';
+
+interface InputProps {
+    onClick?: () => void;
+    className?: string;
+    title: string;
+  }
+
+const Input=({
+    className = '',
+    title,
+  }: InputProps)=> {
+  const [isChecked, setIsChecked] = useState(false);
+
+  return (
+    <>
+      <div className="flex gap-2">
+        <input
+          type="checkbox"
+          checked={isChecked}
+          onChange={() => setIsChecked((prev) => !prev)}
+          title={title}
+          className={'w-4 h-4 text-blue-600 ${className}'}
```

Comment:
You may need to use backtick (`) instead of a single quote to make it a template literal.

```suggestion
          className={`w-4 h-4 text-blue-600 ${className}`}
```

Reply:
I didn't notice. Thank you, have made the changes will also take care from next time.